src/python/pants/task/target_restriction_mixins.py

```diff
@@ -55,6 +55,10 @@ class HasSkipOptionMixin(object):
   def skip_execution(self):
     return self.get_options().skip
 
+  @property
+  def supports_skipping_target_by_tag(self):
```

Comment:
I'd call this `can_skip_individual_targets()`, so that it's not so closely bound to the fact that we happen to use tags to express how we choose which targets to skip.